packages/docs/src/pages/en/getting-started/contributing.md

```diff
@@ -57,14 +65,28 @@ Some of our dependencies use [node-gyp](https://github.com/nodejs/node-gyp#insta
 
 Once you have everything installed, clone the repository:
 
+<v-tabs v-model="tab" color="primary">
```

Comment:
This doesn't need to be fancy the standard code block is more than acceptable as is. The MD files should remain as close to a regular markup language as possible for purposes of translations and maintenance. The other small content changes are fine.

Reply:
Okay, I will switch back then 👍🏼

Reply:
There, this should be resolved.